daemon/state.go

```diff
@@ -269,7 +269,9 @@ func (d *Daemon) regenerateRestoredEndpoints(state *endpointRestoreState) (resto
 			l, _ := labels.FilterLabels(ep.OpLabels.AllLabels())
 			ep.RUnlock()
 
-			identity, _, err := cache.AllocateIdentity(context.Background(), d, l)
+			allocateCtx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
I haven't yet reviewed whether the 2 minute timeout makes sense for all of these cases, but I think that we should consider defining this time somewhere as something like the `pkg/defaults/defaults.go:KVStoreConnectivityTimeout` or something.